aries-site/src/pages/components/anchor.js

```diff
@@ -18,24 +24,154 @@ const Anchor = () => (
+          underline.
         </SubsectionText>
-        <Example
-          details={[
-            `The default anchor color is brand, however, it is possible to use 
-            HPE Design System font colors for anchor in some use cases (i.e. 
-            header and footer). When using an inline text anchor, the color 
-            should remain default.`,
+      </Subsection>
+      <Subsection name="Usage of Anchor vs Button" level={3} gap="small">
+        <SubsectionText>
+          When deciding whether to use Anchor or Button, consider the visual
+          appearance of the element. If the element will look like a tradition
+          browser link, use Anchor. If there will be background or border
+          styling, use{' '}
+          <GrommetAnchor label="Button" href={nameToPath('Button')} />.
+        </SubsectionText>
+        <SubsectionText>
+          Semantically, if you apply an href to either an Anchor or a Button,
+          both will appear as an anchor tag in the DOM. This is why it is
+          important to consider the visual appearance of the element when
+          deciding whether to use Anchor or Button.
+        </SubsectionText>
+        <SubsectionText>
+          Here are some contexts that Anchor should be used:
+        </SubsectionText>
+        <BulletedList
+          items={[
+            'as inline text links',
+            'to add links to resources within contexts such as a form',
           ]}
+        />
+        <SubsectionText>
+          If a clickable element is intended to appear with a background color,
+          border, of hover style, implement this using{' '}
+          <GrommetAnchor label="Button" href={nameToPath('Button')} />.
+        </SubsectionText>
+      </Subsection>
+      <Subsection name="Anchor Labeling" level={3} gap="small">
+        <SubsectionText>
+          It is important that your anchor clearly states what content the
+          anchor leads to. For example, labels such as "Account Information" or
+          "How to change account credentials" are both effective labels that
+          allow the user to have confidence in where the anchor will lead them.
+        </SubsectionText>
+      </Subsection>
+      <Subsection name="Accessibility" level={3} gap="small">
+        <Text weight="bold">
+          Use target and rel props when linking externally:
+        </Text>
+        <SubsectionText>
+          If clicking an Anchor leads to an external URL, apply the these props
+          to the anchor:
+        </SubsectionText>
+        <BulletedList items={['target="_blank"', 'rel="noopener']} />
+        <SubsectionText>
+          The distinction is important to screen reader users to know what's
+          going to happen next. Will I navigate somewhere or will something
+          happen on the page?
+        </SubsectionText>
+        <SubsectionText>
+          Adding the "noopener" property prevents the newly opened page from
```

Comment:
Worthwhile to describe what target="_blank" does?

Reply:
Done.